model/release.go

```diff
@@ -123,7 +132,7 @@ func (r *Release) loadJobs() (err error) {
 		}
 	}()
 
-	jobs := r.manifest["jobs"].([]interface{})
+	jobs := r.manifest.Jobs
 	for _, job := range jobs {
```

Comment:
This can just be `for _, job := range r.manifest.Jobs`

Also, if we define `Jobs` as `[]map[interface{}]interface{}`, we won't need the cast on the next line.